playground/components/header.tsx

```diff
@@ -62,90 +72,146 @@ export const Header: Component<{
-          </svg>
-        </button>
-
-        <button
-          type="button"
-          onClick={() => exportToCsb(props.tabs)}
-          class="px-3 py-2 focus:outline-none focus:ring-1 rounded text-white opacity-80 hover:opacity-100"
-          title="Export to CodeSandbox"
-        >
-          <span class="sr-only">Export to CodeSandbox</span>
-          <svg class="fill-current h-6" preserveAspectRatio="xMidYMid" viewBox="0 0 256 296">
-            <path d="M115 261V154l-91-52v61l42 24v46l49 28zm24 1l51-29v-47l42-25v-60l-93 54v107zm81-181l-49-28-43 24-43-24-49 28 92 53 92-53zM0 222V74L128 0l128 74v148l-128 74L0 222z" />
-          </svg>
-        </button>
-
-        <button
-          type="button"
-          onClick={shareLink}
-          class="px-3 py-2 focus:outline-none focus:ring-1 rounded"
-          classList={{
-            'text-white opacity-80 hover:opacity-100': !copy(),
-            'text-green-100': copy(),
-          }}
-          title="Share with a minified link"
-        >
-          <span class="sr-only">{copy() ? 'Copied to clipboard' : 'Share'}</span>
-          <Icon class="h-6" path={copy() ? link : share} />
+          ) : (
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              class="h-6 w-6"
+              fill="none"
+              viewBox="0 0 24 24"
+              stroke="currentColor"
+            >
+              <path
+                stroke-linecap="round"
+                stroke-linejoin="round"
+                stroke-width={2}
+                d="M4 6h16M4 10h16M4 14h16M4 18h16"
+              />
+            </svg>
```

Comment:
Same as previous comment:
```suggestion
            <Icon path={menu} class="h-6 w-6" />
```

Reply:
Done